perfkitbenchmarker/benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -59,27 +59,27 @@
              '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
              '$FABAN_HOME/search/build/lib/search.jar')
 
-flags.DEFINE_string('client_heap_size', '2g',
+flags.DEFINE_string('websearch_client_heap_size', '2g',
```

Comment:
Sounds good. I definitely don't plan to change the benchmark name, just want to find a balance between:
* Making it clear that changing the flag will only affect cloudsuite web search.
* Making the flag too long.

`cs_websearch` or `websearch` are both fine with me.  Using `faban_heap_size` and `solr_heap_size` is fine, but we would then also need a prefix for `distr`, `ramp_up`, etc.